ext/standard/tests/checkdnsrr.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+bool checkdnsrr ( string $host [, string $type = "MX" ] );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--FILE--
```

Comment:
This needs a SKIPIF section for online tests like @krakjoe mentioned:

if (getenv("SKIP_ONLINE_TESTS")) { die('skip: online test'); }